src/lambda/expression.rs

```diff
@@ -13,118 +13,96 @@ use crate::json::JsonLike;
-          let inp = &input.eval(ctx).await?;
-          Ok(inp.get_path(path).unwrap_or(&async_graphql::Value::Null).clone())
-        }
-        Expression::Literal(value) => Ok(serde_json::from_value(value.clone())?),
-        Expression::EqualTo(left, right) => Ok(async_graphql::Value::from(
-          left.eval(ctx).await? == right.eval(ctx).await?,
-        )),
-        Expression::Unsafe(operation) => {
-          match operation {
-            Operation::Endpoint(req_template) => {
-              let req = req_template.to_request(ctx)?;
-              let url = req.uri().clone();
-              let is_get = req.method() == reqwest::Method::GET;
-              // Attempt to short circuit GET request
-              if is_get {
-                if let Some(cached) = ctx.req_ctx.cache.get(&url) {
-                  return Ok(cached.body);
+                            #[cfg(feature = "unsafe-js")]
+                            {
+                                let input = input.eval(ctx).await?;
+                                result = javascript::execute_js(script, input, Some(ctx.timeout))
+                                    .map_err(|e| EvaluationError::JSException(e.to_string()).into());
+                            }
+                            result
+                        }
+                    }
                 }
-              }
-
-              // Prepare for HTTP calls
-              let res = ctx
-                .req_ctx
-                .execute(req)
-                .await
-                .map_err(|e| EvaluationError::IOException(e.to_string()))?;
-              if ctx.req_ctx.server.enable_http_validation() {
-                req_template
-                  .endpoint
-                  .output
-                  .validate(&res.body)
-                  .map_err(EvaluationError::from)?;
-              }
-
-              // Insert into cache for future requests
-              if is_get {
-                ctx.req_ctx.cache.insert(url, res.clone());
-              }
-
-              Ok(res.body)
             }
-            Operation::JS(input, script) => {
-              let result;
-              #[cfg(not(feature = "unsafe-js"))]
-              {
-                let _ = script;
-                let _ = input;
-                result = Err(EvaluationError::JSException("JS execution is disabled".to_string()).into());
-              }
+        })
+    }
+}
 
-              #[cfg(feature = "unsafe-js")]
-              {
-                let input = input.eval(ctx).await?;
-                result = javascript::execute_js(script, input, Some(ctx.timeout))
-                  .map_err(|e| EvaluationError::JSException(e.to_string()).into());
-              }
-              result
-            }
-          }
-        }
-      }
-    })
-  }
+#[derive(Clone, Debug)]
+pub enum Expression {
+    Unsafe(Operation),
+}
+
+#[derive(Clone, Debug)]
+pub enum Context {
+    Value,
+    Path(Vec<String>),
 }
```

Comment:
Too much whitespace changes. Is it manually performed or some tool was used?

Reply:
I have formatted the code.

Reply:
How did the order of things change automatically?

Reply:
I have utilized a VS Code extension for formatting, and I can undo it if necessary.

Reply:
can you run `./lint.sh` present in the root folder?

Reply:
did